TWEET_COMPONENT.md

```
@@ -0,0 +1,55 @@
+
+## Usage
+
+To embed a tweet in your MDX files:
+
+1. Import the Tweet component at the top of your MDX file:
+
+```mdx
+import Tweet from '../../components/tweet.astro'
+```
+
+2. Use the component with the full tweet URL in the `id` prop:
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)